plugin/sneak.vim

```diff
@@ -18,16 +18,16 @@ func! sneak#init()
   unlockvar g:sneak#opt
   "options                                 v-- for backwards-compatibility
   let g:sneak#opt = { 'f_reset' : get(g:, 'sneak#nextprev_f', get(g:, 'sneak#f_reset', 1))
-      \ ,'t_reset'      : get(g:, 'sneak#nextprev_t', get(g:, 'sneak#t_reset', 1))
-      \ ,'s_next'       : get(g:, 'sneak#s_next', 0)
-      \ ,'absolute_dir' : get(g:, 'sneak#absolute_dir', 0)
-      \ ,'textobject_z' : get(g:, 'sneak#textobject_z', 1)
-      \ ,'use_ic_scs'   : get(g:, 'sneak#use_ic_scs', 0)
-      \ ,'map_netrw'    : get(g:, 'sneak#map_netrw', 1)
-      \ ,'streak'       : get(g:, 'sneak#streak', 0) && (v:version >= 703) && has("conceal")
-      \ ,'streak_esc'   : get(g:, 'sneak#streak_esc', "\<space>")
-      \ ,'prompt'       : get(g:, 'sneak#prompt', '>')
-      \ }
+        \ ,'t_reset'      : get(g:, 'sneak#nextprev_t', get(g:, 'sneak#t_reset', 1))
+        \ ,'s_next'       : get(g:, 'sneak#s_next', 0)
+        \ ,'absolute_dir' : get(g:, 'sneak#absolute_dir', 0)
+        \ ,'textobject_z' : get(g:, 'sneak#textobject_z', 1)
+        \ ,'use_ic_scs'   : get(g:, 'sneak#use_ic_scs', 0)
+        \ ,'map_netrw'    : get(g:, 'sneak#map_netrw', 1)
+        \ ,'streak'       : get(g:, 'sneak#streak', 0) && (v:version >= 703) && has("conceal")
+        \ ,'streak_esc'   : get(g:, 'sneak#streak_esc', "\<space>")
+        \ ,'prompt'       : get(g:, 'sneak#prompt', '>')
+        \ }
```

Comment:
That just happened accidentally. I have a leader command to indent the whole file, and apparently some of my indentation settings are a little different from yours.